pkg/util/nsenter/nsenter.go

```diff
@@ -66,7 +66,7 @@ type Nsenter struct {
 }
 
 // NewNsenter constructs a new instance of Nsenter
-func NewNsenter() *Nsenter {
+func NewNsenter() (*Nsenter, error) {
```

Comment:
I think we should also check for presence of `nsenter` by adding it at L82

Reply:
make sense.